_articles/using-retries.md

```diff
@@ -0,0 +1,166 @@
+# Practical Circuit Breaking and Retries
+
+Circuit breaking and automatic retries are two powerful features of Envoy, and this article gives you a few examples of how to configure and use them.
+
+## The setup
+
+The Envoy documentation provides a good overview of
+[how to run the example](https://www.envoyproxy.io/docs/envoy/latest/start/sandboxes/zipkin_tracing)
+
+You should already have the following installed from running Envoy on your laptop:
+
+- [Docker](https://docs.docker.com/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+- [Git](https://help.github.com/articles/set-up-git/)
+- [curl](https://curl.haxx.se/)
+
+Additionally, install this service to drive load to our examples:
+
+- [wrk](https://github.com/wg/wrk)
+
+## Configuring latency and success rate
+
+In our first example we've add a configurable latency and success rate to the
+Python service that underlies all the Envoy examples.
+
+First, check out this tag of the example repo:
+
+`https://github.com/turbinelabs/envoy-examples/tree/step1`
+
+Next, start the Zipkin tracing example in the `zipkin-tracing` directory by running:
+
+`docker-compose up --build -d`
+
+Run wrk with:
+
+`wrk -c 1 -t 1 --latency -d 5s http://localhost:8000/service/1`
+
+which returns, for example:
+
+```console
```

Comment:
use shell instead of console for output; console is reserved for the copy/pastable stuff

Reply:
(and elsewhere in this doc)